crypto-ffi/src/generic.rs

```diff
@@ -719,13 +721,27 @@ pub trait CoreCryptoCallbacks: std::fmt::Debug + Send + Sync {
     ) -> bool;
 }
 
+static INIT_LOGGER: Once = Once::new();
+static LOGGER: LazyLock<ReloadLog<CoreCryptoLoggerWrapper>> = LazyLock::new(|| {
```

Comment:
This `ReloadLog` struct has a blocking lock inside. Are we sure we want to use it?

Reply:
As discussed offline this should be fine.